### DISCUSSION TITLE
关于Github图床中删除B图片导致A图片无法访问

### DISCUSSION THREAD

**Discussion by starmilkxin** [Molunerfinn/PicGo#784] [Q&A]
请问为什么我在Github图床中删除B图片后，会导致原来可以通过url访问的A图片无法访问了呢?
----------------------------------------------
好像是因为私有仓库的原因，换成公共仓库后就没有这个问题了

**Comment by Molunerfinn** [comment]
私有仓库本身就不支持公开访问
